.github/workflows/run_tests.yaml

```diff
@@ -8,14 +8,30 @@ on:
       - "*"
 
 jobs:
-  running-tests:
+  build:
     uses: kaltura/ovp-pipelines-pub/.github/workflows/player_tests.yaml@master
     with:
-      yarn-run-to-execute: 'build eslint flow test'
+      node-version: '20.x'
```

Comment:
why there cannot be 1 job and then call all of them within `yarn-run-to-execute`?